packages/core/test/smoke.test.ts

```diff
@@ -13,10 +13,10 @@ function ignoreErrors(cb: () => void) {
 
 test("server vars should not be prefixed", () => {
   ignoreErrors(() => {
+    // @ts-expect-error - server should not have FOO_ prefix
```

Comment:
this is suboptimal - would love to find a solution where the error is kept on the invalid key as it was, and not the entire object like it is now :/

Reply:
actually never mind it didn't work at all :/